### ISSUE TITLE
USB connector interference

### ISSUE THREAD

**Issue by charlespax** [PaxInstruments/labwiz-board#111]
The USB connector interferes with the enclosure. Find a more suitable connector. Maybe move the connector.

**Comment by charlespax** [comment]
There does exist a better connector. In the HQB markets I've seen a flangeless version of the connector in the T400. It looks more or less like this...
![screen shot 2017-01-22 at 12 14 23](https://cloud.githubusercontent.com/assets/281642/22179941/6cef6e98-e09c-11e6-81e3-b998b1a38d12.png)

I have to find the part and implement the footprint. I believe one of the Blinkin Labs products uses it. https://github.com/Blinkinlabs

**Comment by electrokean** [comment]
Looks like the same footprint at the Molex 105017-0001 or FCI 10118194-0001LF
I've found them much less likely to be pulled off the PCB than the flanged versions

**Comment by electrokean** [comment]
Hang on - isn't this the footprint you referenced in https://github.com/PaxInstruments/labwiz-board/issues/18

**Comment by charlespax** [comment]
Nope. #18 is about the footprint and the internal layers. This issue (#111) is about the flange of the MicroUSB connector interfering with the enclosure.

This part is similar to what is on the T400. Check out the flange.
![screen shot 2017-01-23 at 12 32 20](https://cloud.githubusercontent.com/assets/281642/22191657/01ed183a-e168-11e6-8fbc-b0a146ff45c0.png)

**Comment by electrokean** [comment]
Aha, I see.  Not the mounting flange, but the "entry" flange.
Fot the Molex part, that would be 105017-1001
http://www.digikey.com/product-detail/en/molex-llc/1050171001/WM11262CT-ND/3313406
